client/flutter/lib/api/content/schema/index_content.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/material.dart';
+import 'package:who_app/api/content/content_bundle.dart';
+import 'package:who_app/api/linking.dart';
+import '../content_loading.dart';
```

Comment:
nit: use the `package:who_app/<path>` syntax instead of relative paths for consistency

Reply:
fixed in full repo